backend/src/utils/userTests/userTests.ts

```diff
@@ -0,0 +1,27 @@
+import { IAnswer, TCorrectAnswers } from 'interfaces/Ientities/Itest';
+
+const checkTestResults = (
+  userAnswers: IAnswer[],
+  trueAnswers: TCorrectAnswers<IAnswer, 'aN', 'correctAnswer'>[],
+) => {
+  const userAnswersSorted = [...userAnswers].sort(
+    (currAnswer, nextAnswer) => currAnswer.qN - nextAnswer.qN,
+  );
+  const wrongAnswers = [];
+  for (let i = 0; i < userAnswersSorted.length; i += 1) {

```

Comment:
`i++` or `++i` looks much better

Reply:
the reason is simple we just have this rule https://eslint.org/docs/rules/no-plusplus